iframe/style.css

```diff
@@ -327,9 +343,6 @@ body.stop h1 {
 
 body.slow h1,
 body.without h1 {
-  background: linear-gradient(#39ebfd, #6b74fb);
```

Comment:
Perhaps a [feature query](https://developer.mozilla.org/en-US/docs/Web/CSS/@supports) could be used to enable the gradient if it's supported?